src/Core/Checkout/Cart/Order/RecalculationService.php

```diff
@@ -409,4 +373,14 @@ private function recalculateCart(Cart $cart, SalesChannelContext $context): Cart
             return $validated->getCart();
         });
     }
+
+    private function getOrderConversionContext(bool $shouldIncludeDeliveries = false): OrderConversionContext
+    {
+        return (new OrderConversionContext())
+            ->setIncludeCustomer(false)
+            ->setIncludeBillingAddress(false)
+            ->setIncludeDeliveries($shouldIncludeDeliveries)
```

Comment:
I think it's not necessary to wrap this, let a consumer call this, e.g. `$this->getOrderConversionContext()->setIncludeDeliveries(false)` (as true is default):
```suggestion
    private function getOrderConversionContext(): OrderConversionContext
    {
        return (new OrderConversionContext())
            ->setIncludeCustomer(false)
            ->setIncludeBillingAddress(false)
```

Reply:
True, I was trying to avoid the same changed object in each method.